test/testcanvas.go

```diff
@@ -167,6 +168,32 @@ func (c *testCanvas) Capture() image.Image {
 	return img
 }
 
+// CanvasItem is a part of a canvas and returned by InspectCanvasItems().
+type CanvasItem struct {
```

Comment:
Is it worth adding a new type for this? The resulting is string comparison instead of compile time type checks - in the same number of lines of code.